batch_import/EXAMPLE.md

```diff
@@ -135,48 +135,48 @@ $ brew install curl
+$ curl https://storage.googleapis.com/camio_test_general/video_importer_test_files.zip -o video_importer_test_files.zip
   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                  Dload  Upload   Total   Spent    Left  Speed
-100  303M  100  303M    0     0  2233k      0  0:02:19  0:02:19 --:--:-- 1647k
-$ unzip batch_import_video_files.zip
-Archive:  batch_import_video_files.zip
-   creating: input_videos/
-  inflating: input_videos/office_street-camera-1495068758.mp4
-  inflating: input_videos/office_street-camera-1495069946.mp4
+ 13  236M   13 30.8M    0     0  3576k      0  0:01:07  0:00:08  0:00:59 3747k
+$ unzip video_importer_test_files.zip
+Archive:  video_importer_test_files.zip
+   creating: video_importer_test_files/
+  inflating: video_importer_test_files/C2_Hi20161009-120237-1476014557.mp4
+  inflating: video_importer_test_files/C2_Hi20161009-131237-1476018757.mp4
 ```
 
-You will now have a directory `input_videos` that contians the two files:
+You will now have a directory `video_importer_test_files` that contians the two files:
 
 ```
-office_street-1495068758.mp4
-office_street-1495069946.mp4
+C2_Hi20161009-120237-1476014557.mp4 
+C2_Hi20161009-131237-1476018757.mp4
 ```
 
 ##### Constructing the File-Parsing Regex
 
 You will be running the video-import script over a directory of video files. For this example we will assume you are using the example video files
-[supplied by Camio](https://storage.googleapis.com/camio_test_general/batch_import_video_files.zip). Downloading and unzipping these files was described
-in the [last section](#get-the-testing-videos)
+[supplied by Camio](https://storage.googleapis.com/camio_test_general/video_importer_test_files.zip). Downloading and unzipping these files was described
+in the [last section](#get-the-testing-videos).
 
-As described in the previous section, you now have a directory `input_videos` that contians the two files:
+As described in the previous section, you now have a directory `video_importer_test_files` that contians the two files:
 
 ```
-office_street-1495068758.mp4
-office_street-1495069946.mp4
+C2_Hi20161009-120237-1476014557.mp4 
+C2_Hi20161009-131237-1476018757.mp4
 ```
 
 Then you would use the following string as the regular expression passed to the video-import script.
 
 * On Linux and OSX *
 
-`.*/(?P<camera>\w+?)\-(?P<epoch>\d+)\.mp4`
+`.*/(?P<camera>\w+?)\-.*\-(?P<epoch>\d+)\.mp4`
 
 * On Windows *
 
-`.*[\\](?P<camera>\w+?)\-(?P<epoch>\d+)\.mp4`
+`.*[\\](?P<camera>\w+?)\-.*\-(?P<epoch>\d+)\.mp4`
 
-This captures the `office_street` part as the camera name (used for registering the camera with Camio), and parses the `1495068758` part as the Unix-timestamp
+This captures the `C2_Hi20161009` part as the camera name (used for registering the camera with Camio), and parses the `1476014557` part as the Unix-timestamp
```

Comment:
@jhallard `C2_Hi` is the camera name (camera 2 high resolution stream), so modify the regex to avoid including the date in the camera name.